MiniSim/Menu.swift

```diff
@@ -116,9 +105,7 @@ class Menu: NSMenu {
     private func assignKeyEquivalent(devices: [NSMenuItem]) {
         for (index, item) in devices.enumerated() {
             if index > maxKeyEquivalent {
-                DispatchQueue.main.async {
-                    item.keyEquivalent = ""
-                }
+                item.keyEquivalent = ""
```

Comment:
Can you leave this DispatchQueue call in order to prevent breaking this on older OS versions?